manifest.json

```diff
@@ -11,6 +11,9 @@
   "scripts": {
     "prereleasy": "bash lint.sh"
   },
+  "peerDependencies": {
```

Comment:
Are we using this dependency here?

Reply:
No, it was a test for the megamenu, but it doesn't work. Already eliminated